src/freertos_drivers/ti/CC32xxSPI.cxx

```diff
@@ -31,21 +31,27 @@
  * @date 30 May 2016
  */
 
+#include "CC32xxSPI.hxx"
+
 #include <algorithm>
+#include <unistd.h>
 
+#include "inc/hw_gpio.h"
 #include "inc/hw_types.h"
 #include "inc/hw_memmap.h"
 #include "inc/hw_ints.h"
+#include "inc/hw_udma.h"
+#include "driverlib/gpio.h"
 #include "driverlib/rom.h"
 #include "driverlib/rom_map.h"
 #include "driverlib/spi.h"
 #include "driverlib/interrupt.h"
 #include "driverlib/prcm.h"
 
-#include "CC32xxSPI.hxx"
+#include <ti/drivers/dma/UDMACC32XX.h>
```

Comment:
I'm getting the following compile error here:
```
/home/bracz/train/openlcb/tcs_booster/deps/tcs_throttle/deps/openmrn/src/freertos_drivers/ti/CC32xxSPI.cxx:51:39: fatal error: ti/drivers/dma/UDMACC32XX.h: No such file or directory
 #include <ti/drivers/dma/UDMACC32XX.h>
```
This is probably with the CC3200 SDK.